templates/resources/ec2_transit_gateway_route_table_association.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### First example
```

Comment:
(nit): We can remove the `### First example` unless we intend to add any more to the PR. The statement below would help deliver the usecase.

Reply:
Removed